pkg/config/raw.go

```diff
@@ -14,19 +14,19 @@
 # limitations under the License.
 **/
 
-package engine
+package config
```

Comment:
Why has this been moved from `engine` to `config`

Reply:
Because this represents a `Raw` config. Importing `enging.Raw` does not make the intent as clear as `config.Raw` when imported.